src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/TensorPrimitives.netcore.cs

```diff
@@ -1220,102 +1222,892 @@ private static float MinMaxCore<TMinMaxOperator>(ReadOnlySpan<float> x)
 
             ValidateInputOutputSpanNonOverlapping(x, destination);
 
+            // Since every branch has a cost and since that cost is
```

Comment:
Is the intent to do the other methods in this PR, or are you planning to do them all in a subsequent PR?

Reply:
Subsequent PR. Wanted to get the initial code pattern up before doing “everything”